Snowflake.Data/Core/Authenticator/OktaAuthenticator.cs

```diff
@@ -88,7 +88,10 @@ async Task IAuthenticator.AuthenticateAsync(CancellationToken cancellationToken)
                     _rawSamlTokenHtmlString = await samlRawResponse.Content.ReadAsStringAsync(cancellationToken).ConfigureAwait(false);
 #endif
                     s_logger.Debug("step 5: Verify postback URL in SAML response");
```

Comment:
Can we add debug info about skippig this step?

Reply:
Sure, added a log when the check is skipped